### PR TITLE
update views auditcare logs

### PR DESCRIPTION
**Pull request by emord**
Seems like two apps we should be logging? @krsdimagi 

buddy @gcapalbo

### CODE REVIEW

File: settings.py
Comment (emord) [inline settings.py]: `hqadmin` we can view user's restores

File: settings.py
Comment (emord) [inline settings.py]: `accounting` can make changes to domains subscriptions and stuff

**Review by gcapalbo — APPROVED** [review]

**Comment by gcapalbo** [comment]
@emord were you waiting for @krsdimagi to confirm before we merge?

**Comment by emord** [comment]
@gcapalbo I thought he may have some thoughts or why/if we should/shouldn't be logging these. feels fine to merge to me though?

**Comment by krsdimagi** [comment]
@emord @gcapalbo hey, looks good to me.  I think I err on the side of logging more rather than less.  Good to merge.

**Pull request merged**
